### PR TITLE
sci-biology/aghermann: make it repoman clean

### PR DESCRIPTION
**Pull request by hmmr**

### CODE REVIEW

File: sci-biology/aghermann/ChangeLog [hunk 1]
```diff
@@ -2,11 +2,5 @@
...
 
-  18 Feb 2015; Justin Lecher <jlec@gentoo.org> aghermann-1.0.3.ebuild:
-  sci-biology/aghermann: Fix slot operators
-
-  18 Feb 2015; Justin Lecher <jlec@gentoo.org> aghermann-1.0.3.ebuild:
-  sci-biology/aghermann: Fix indention and use virtual for pkgconfig
-
-  17 Jan 2015; Andrei Zavada <johnhommer@gmail.com> aghermann-1.0.3.ebuild:
-  Initial packaging for gentoo.
+  19 Feb 2015; Andrei Zavada <johnhommer@gmail.com> aghermann-1.0.3.ebuild:
```
Comment (jlec) [inline sci-biology/aghermann/ChangeLog]: No need to remove all ChangeLog entries ;)

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 2]
```diff
@@ -12,17 +12,16 @@ LICENSE="GPL-2+"
...
+RDEPEND="dev-lang/lua:*
+	 dev-libs/libconfig
+	 dev-libs/libunique:3
+	 media-libs/libsamplerate
+	 sci-libs/fftw:3.0
+	 sci-libs/gsl
+	 sci-libs/itpp
+	 x11-libs/gtk+:3
+	 x11-libs/vte:2.90"
+
```
Comment (jlec) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Please use only tab for indention. 

https://devmanual.gentoo.org/ebuild-writing/file-format/ > "Indenting and Whitespace"

File: sci-biology/aghermann/ChangeLog (hunk 1 above)
Comment (hmmr) [inline sci-biology/aghermann/ChangeLog]: Sorry, that was really accidental. During a rebase, I knew there should be a conflict on this file, and blindly went with the local version. It just didn't occur to me that someone did my work while I was busy sorting out the other PR.

**Comment by hmmr** [comment]
Thanks! Three beers to everyone on me!

**Comment by jlec** [comment]
Just go on. Some minor things to improve and we are happy to give you direct access to the overlay.

**Pull request merged**